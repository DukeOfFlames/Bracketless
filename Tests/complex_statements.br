START

if true {
    #drucke{"if 1"}
    #drucke{"if 2"}
}

if false {
    #drucke{"if-else 1"}
    #drucke{"if-else 2"}
} else {
    #drucke{"if-else 3"}
    #drucke{"if-else 4"}
}

for n : {#range{5}} {
    #drucke{"for " + #string{n} + " 1"}
    #drucke{"for " + #string{n} + " 2"}
}

Â°x = 4.0
while {x > 0.4} {
    #drucke{"while " + #string{x} + " 1"}
    #drucke{"while " + #string{x} + " 2"}
    x = x / 3
}

END