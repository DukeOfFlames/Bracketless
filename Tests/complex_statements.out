"if 1"
"if 2"
"if-else 3"
"if-else 4"
"for 0 1"
"for 0 2"
"for 1 1"
"for 1 2"
"for 2 1"
"for 2 2"
"for 3 1"
"for 3 2"
"for 4 1"
"for 4 2"
"while 4.0 1"
"while 4.0 2"
"while 1.3333333333333333 1"
"while 1.3333333333333333 2"
"while 0.4444444444444444 1"
"while 0.4444444444444444 2"
